### ISSUE TITLE
[Civics] Query number of players in Demographic

### ISSUE THREAD

**Issue by Dave247** [StrangeLoopGames/EcoSuggestions#1703]
There should be a way to query the number of players in a demographic in a law, and I'm very surprised that there isn't. I found someone in the official Eco Discord ask about this back in 2020 during 9.1, but sadly it doesn't seem like anything has been added since then.
https://canary.discord.com/channels/254025510651297802/469297188909088789/771424521135063081

**Comment by Dave247** [comment]
For now this has been solved by [Nid's Toolbox Light Laws Module](https://forum.play.eco/filebase/file/380-nidtoolbox-laws-module/), however I really think this should be part of the base game.

**Comment by Sarah-Graesyn** [comment]
I am pretty sure that this is possible with the law system as it exists without any form of modding. In fact I am quite certain that if you hover your mouse over the name of the Demographic in game it creates a pop-up that tells you exactly how many people are in it as well as the ability to click the demographic and find out the players in said demographic.

**Comment by Dave247** [comment]
> I am pretty sure that this is possible with the law system as it exists without any form of modding. In fact I am quite certain that if you hover your mouse over the name of the Demographic in game it creates a pop-up that tells you exactly how many people are in it as well as the ability to click the demographic and find out the players in said demographic.

Yet the problem is there is no way to test that value as a condition within a law itself, which is my original problem. The math already clearly exists, it just needs to made into a law conditional as done in the linked mod.

**Comment by Dave247** [comment]
The intent was to have demographics act as a way of grouping players into essentially separate towns, by using a custom chat message as the trigger to add oneself to a particular side, with an added condition to test the current number of players in a demographic against another demographic as a form of balancing, to make sure that one didn't have more players then the other.
This scenario will become redundant come with version 10 with the introduction of the settlement system providing a much better method for it, though there still may well be other use case scenarios for it.

To summarise, the intended function is to test the value of the current amount of players in a demographic against something else, such as another demographic using only civic blocks in-game. Though really it could be used to test against anything, though I admittedly cannot think of any other scenarios at present.
